lib/screens/home.dart

```diff
@@ -35,10 +35,12 @@
 library;
 
 import 'package:flutter/material.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
 
 import 'package:intl/intl.dart';
 import 'package:solidpod/solidpod.dart';
-
+import 'package:solidpod/src/widgets/setting.dart';
```

Comment:
Do not reference src files from a package.

These should be exported from solidpod.dar!

Also they are in the wrong ordering here according to the style guide.